### PR TITLE
Use regular NPM registry for snaps on Firefox

### PR DESCRIPTION
**Pull request by FrederikBolding**

### CODE REVIEW

**Comment by kumavis** [comment]
deps audit fix is [this](https://github.com/MetaMask/metamask-extension/pull/14437), but blocked by the issue solved here. may need to merge these PRs to unblock ourselves

**Review by kumavis — APPROVED** [review]

File: app/scripts/metamask-controller.js
Comment (shanejonas) [inline app/scripts/metamask-controller.js:620]:
```suggestion
```

**Review by ryanml — APPROVED** [review]

**Review by shanejonas — APPROVED** [review]

**Pull request merged**
